README.md

```diff
@@ -177,8 +183,8 @@ less: {
     options: {
       paths: ["assets/css"],
       plugins: [
-        new (require('less-plugin-autoprefix'))({browsers: ["last 2 versions"]}),
-        new (require('less-plugin-clean-css'))(cleanCssOptions)
+        new require('less-plugin-autoprefix')({browsers: ["last 2 versions"]}),
```

Comment:
Looks like this change is already in on gruntjs' master. 
( to be honest, I think this change in readme came from my compiling what I forked, I didn't make this change )